docker/docker-compose.yaml

```diff
@@ -36,83 +36,82 @@ services:
   pocs-config-server:
     image: huntsmanarray/huntsman-pocs:develop
     tty: true
+    init: true
     container_name: pocs-config-server
     hostname: pocs-config-server
-    privileged: true
+    deploy:
+      mode: global
     network_mode: host
     environment:
-      HUNTSMAN_POCS: /var/huntsman
+      HUNTSMAN_POCS: /huntsman
       PANOPTES_CONFIG_HOST: 0.0.0.0
       PANOPTES_CONFIG_PORT: 6563
     restart: on-failure
     volumes:
-      - config:/var/huntsman/huntsman-pocs/conf_files
-      - panlog:/var/huntsman/logs
-      - images:/var/huntsman/images
-    command: [ "panoptes-config-server --verbose run --no-save-local --no-load-local --config-file /var/huntsman/huntsman-pocs/conf_files/huntsman.yaml" ]
+      - config:/huntsman/conf_files
+      - panlog:/huntsman/logs
+      - images:/huntsman/images
+    command: [ "panoptes-config-server --verbose run --no-save-local --no-load-local --config-file /huntsman/conf_files/huntsman.yaml" ]
 
   pyro-name-server:
     image: huntsmanarray/huntsman-pocs:develop
     tty: true
+    init: true
     container_name: pyro-name-server
     hostname: pyro-name-server
+    deploy:
+      mode: global
     privileged: true
     network_mode: host
     restart: on-failure
-    depends_on:
-      - "pocs-config-server"
     environment:
-      HUNTSMAN_POCS: /var/huntsman
-      PANDIR: /var/panoptes
-      POCS: /var/panoptes/POCS
+      HUNTSMAN_POCS: /huntsman
       PANOPTES_CONFIG_PORT: 6563
-    command: [ "/var/panoptes/POCS/scripts/wait-for-it.sh localhost:6563 -- huntsman-pyro nameserver --auto-clean 90" ]
+    command: [ "wait-for-it localhost:6563 -- huntsman-pyro nameserver --auto-clean 90" ]
     volumes:
-      - config:/var/huntsman/huntsman-pocs/conf_files
-      - panlog:/var/huntsman/logs
-      - images:/var/huntsman/images
+      - config:/huntsman/conf_files
+      - panlog:/huntsman/logs
+      - images:/huntsman/images
 
   pocs-control:
     image: huntsmanarray/huntsman-pocs:develop
     tty: true
+    init: true
+    stdin_open: true
     container_name: pocs-control
     hostname: pocs-control
+    deploy:
+      mode: global
     privileged: true
     network_mode: host
     restart: on-failure
-    depends_on:
-      - "pyro-name-server"
     environment:
-      HUNTSMAN_POCS: /var/huntsman
-      PANDIR: /var/panoptes
-      POCS: /var/panoptes/POCS
+      HUNTSMAN_POCS: /huntsman
       PANOPTES_CONFIG_PORT: 6563
-    command: [ "/var/panoptes/POCS/scripts/wait-for-it.sh localhost:6563 -- tail -f /dev/null" ]
+    command: [ "wait-for-it localhost:6563 -- tail -f /dev/null" ]
```

Comment:
What's this one all about?

Reply:
It just keeps the machine running so you can do an `exec` and connect.  But we can chat about this and if we want to start an `ipython` session by default.